bluebottle/time_based/views.py

```diff
@@ -447,33 +447,36 @@ class DateParticipantExportView(PrivateFileView):
     def get(self, request, *args, **kwargs):
         activity = self.get_object()
 
-        response = HttpResponse()
-        response['Content-Disposition'] = 'attachment; filename="participants.csv"'
-        response['Content-Type'] = 'text/csv'
+        filename = 'participants for {}.xlsx'.format(activity.title)
+        workbook = xlsxwriter.Workbook(filename, {'remove_timezone': True})
+        worksheet = workbook.add_worksheet()
 
-        writer = csv.writer(response)
         slots = activity.active_slots.order_by('start')
         row = [field[1] for field in self.fields]
         for slot in slots:
-            if slot.location and not slot.is_online:
-                tz = timezone(slot.location.timezone)
-            else:
-                tz = get_current_timezone()
-            start = slot.start.astimezone(tz)
-
+            start = slot.start.replace(tzinfo=None)
             row.append("{}\n{}".format(slot.title or str(slot), start.strftime('%d-%m-%y %H:%M')))
-        writer.writerow(row)
+        worksheet.write_row(0, 0, row)
+        t = 0
```

Comment:
Instead of keeping track of the index manually, do:
```
 for t, participant in enumerate(activity.contributors.instance_of(
￼            DateParticipant
￼)):